src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayRent = 40;
+  let sumForRent = 0;
+
+  if (days >= 7) {
+    sumForRent = oneDayRent * days - 50;
+
+    return sumForRent;
+  }
+
+  if (days >= 3) {
+    sumForRent = oneDayRent * days - 20;
+
+    return sumForRent;
```

Comment:
As with the previous return, the `else` is not needed here either. Simplify the code by removing it.